src/Adapter/PublicCDNAdapter.php

```diff
@@ -15,9 +16,12 @@ class PublicCDNAdapter extends PublicAdapter implements SilverstripePublicAdapte
 {
     protected $cdnPrefix;
 
-    public function __construct(S3Client $client, $bucket, $prefix = '', $cdnPrefix = '', array $options = [])
+    protected $cdnAssetPath;
```

Comment:
A suggestion to change in all occurrences possibly better expressing what this config is for?

```suggestion
    protected $cdnAssetsDir;
```